java/test/src/main/java/org/ray/api/test/RayConfigTest.java

```diff
@@ -13,7 +13,6 @@ public void testCreateRayConfig() {
     System.setProperty("ray.home", "/path/to/ray");
     RayConfig rayConfig = RayConfig.create();
 
-    Assert.assertEquals(rayConfig.rayHome, "/path/to/ray");
```

Comment:
why remove this?

Reply:
This `assert` can't pass in my machine.
It always be the path setted [here](https://github.com/ant-tech-alliance/ray/blob/simplify_config/java/test/src/main/java/org/ray/api/test/TestListener.java#L12).

Maybe we can remove this line`System.setProperty("ray.home", "/path/to/ray");`,
and rephrase the assert to`assert(rayConfig.rayHome, "../..")`;